### PR TITLE
Add Dataplane Telemetry PTF tests and required infra

### PR DESCRIPTION
**Pull request by shruthi9**
Corresponding test plan reviewed with the community:
https://github.com/Azure/SONiC/pull/182/files#diff-b5c45266221678f1147e4119b8a12209

### CODE REVIEW

File: ansible/roles/test/tasks/dtel.yml [hunk 1]
```diff
@@ -0,0 +1,137 @@
+#### This playbook tests Dataplane Telemetry (DTel)
+- block:
+    - fail: msg="Please set ptf_host variable"
+      when: ptf_host is not defined
+
+    - name: Ensure LLDP Daemon stopped
+      become: yes
```
Comment (kperumalbfn) [inline ansible/roles/test/tasks/dtel.yml:7]: Do we need to stop/start the LLDP daemon for Dtel?

File: ansible/roles/test/files/ptftests/dtel/tests/dtel_drop.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+from ptf.thriftutils import *
+from switch_ptf_config import *
+
+mac_all_ospf_routers = '01:00:5e:00:00:05'
+ipaddr_all_ospf_routers = '224.0.0.5'
+
+min_sleeptime = 3
+
+################################################################################
+@group('drop')
```
Comment (kperumalbfn) [inline ansible/roles/test/files/ptftests/dtel/tests/dtel_drop.py:25]: We can remove these PTF groups from all ptf scripts.

File: ansible/roles/test/vars/testcases.yml [hunk 3]
```diff
@@ -58,6 +58,11 @@ testcases:
       topologies: [t0, t0-64, t0-64-32, t0-116]
       required_vars:
           ptf_host:
+    dtel:
+      filename: dtel.yml
+      topologies: [ptf32]
```
Comment (kperumalbfn) [inline ansible/roles/test/vars/testcases.yml:63]: How about ptf64 topology?

**Review by kperumalbfn — APPROVED** [review]
Reviewed ansible yml files.

File: ansible/roles/test/vars/testcases.yml (hunk 3 above)
Comment (shruthi9) [reply ansible/roles/test/vars/testcases.yml:63]: Dataplane telemetry tests require only 3 ports for now. So, we use only ptf32 topology

**Pull request merged**